static/figure/js/models/figure_model.js

```diff
@@ -104,7 +110,7 @@
 
             return json;
         },
-
+        
```

Comment:
@dominikl I wonder if you could remove this whitespace change (and see if there are any others that could be cleaned up)?
I don't know if you're using ```$ git add -p``` but I find this can help avoiding unnecessary whitespace changes?
Thanks